app/models/activity_log_event.rb

```diff
@@ -0,0 +1,15 @@
+class ActivityLogEvent < Audited::Audit
```

Comment:
Nice to see! I think it would be much cleaner to accept an `audit` in the constructor than to inherit here? Then we actually control the interface of this object.